src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -209,8 +209,16 @@ MenuItemList NotationContextMenuModel::makeElementItems()
 
     if (hitElement) {
         QString pageCode = EditStyle::pageCodeForElement(hitElement);
+
         if (!pageCode.isEmpty()) {
-            item->setArgs(mu::actions::ActionData::make_arg1<QString>(pageCode));
+            if (pageCode == "text-styles") {
```

Comment:
To prevent having `"text-styles"` written as a string here (which is typo-prone or might get out of sync in the future), I think we can just get rid of it. Checking `if (!subPageCode.isEmpty())` is enough.